scripts/hail_batch/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -0,0 +1,62 @@
+"""Launch analysis runner for all cell types and chromosomes"""
+
+import re
+import os
+from analysis_runner.cli_analysisrunner import run_analysis_runner
```

Comment:
I'm getting a `ModuleNotFoundError` for this?

Reply:
Can you send me a hail batch link where you're seeing this ModuleNotFoundError, or if you're running it locally, make sure to install the analysis-runner CLI `conda install -c cpg analysis-runner`.

Reply:
This only came up in my commit hook. Once I run it, I can send you a link (analysis runner is installed in my conda environment)

Reply:
I just needed to update the analysis runner